tests/scripts/common/common_functions.sh

```diff
@@ -847,8 +847,9 @@ function create_expected_liststacktunables_json() {
 
 		printf '\n    "hpo_algo_impl":  "'${hpo_algo_impl}'",' >> ${file_name}
 
-		printf '\n    "deployment_name": "'${deployment_names[index]}'",' >> ${file_name}
-		printf '\n    "namespace":  '$(cat ${autotune_json} | jq '.metadata.namespace')',' >> ${file_name}
+		# Uncomment these when the json output is updated
```

Comment:
Why is this file included? @ShishirH - Can you please check and re-base it so that it includes only the relevant files.

Reply:
Sorry for that! Removed the file and rebased it